src/telliot_feeds/reporters/tellor_360.py

```diff
@@ -235,3 +236,32 @@ async def fetch_datafeed(self) -> Optional[DataFeed[Any]]:
                 self.autopaytip = await self.rewards()
                 return self.datafeed
         return None
+
+    async def fetch_gas_price(self, speed: Optional[str] = None) -> Optional[int]:
+        mainnet = (1, 5)
```

Comment:
Instead of using `if/else` for this in those chains, I'd make a `fetch_matic_gas_price` & `fetch_ethereum_gas_price` function. For reusability. Also cleaner. Maybe you'd want to make those hardcoded ethereum chains into a constant at a lower level in the package that can be reused elsewhere..

Then the more general `fetch_gas_price` function could take a chain id and select more specific gas fetchers based on if that chain is in the above constant (tuple)

Reply:
How would that work then in fetch_gas_price you take in chain_id and based on that if mainnet use ```fetch_ethereum_gas``` and if polygon use ```fetch_matic_gas``` since they all use the same reporter script?

Reply:
I'm saying it might be good to have the logic for each as functions to be reused elsewhere

Reply:
Sounds good